scale/docs/architecture/jobs/resources.rst

```diff
@@ -0,0 +1,54 @@
+
+.. _architecture_jobs_resources:
+
+Resources
+========================================================================================================================
+
+The resources document is a JSON document that defines a set of named resources with floating point values.
+
+**Example job configuration:**
+
+.. code-block:: javascript
+
+  {
+     "version": "1.0",
+     "resources": {
+        "cpus": 1.5,
```

Comment:
This isn't just used for the custom resources (which is a field for job types). This is the general JSON schema to use for talking about any resources. It will include the defined cpus, mem, etc as well as custom resources potentially. This format is stored in the queue and job_exe models to represent the resources required/scheduled. It is used in job_type.custom_resources for just custom stuff so we don't impact the existing columns. The job_type fields will go through changes as Seed comes in.